goalie/go_mesh_seq.py

```diff
@@ -228,6 +232,17 @@ def indicate_errors(
                     indi.interpolate(abs(indi))
                     self.indicators[f][i][j].interpolate(ufl.max_value(indi, 1.0e-16))
 
+            # discard extra subinterval solution field
+            self.solutions[f][FWD_OLD].pop()
+            self.solutions[f][ADJ_NEXT].pop()
```

Comment:
Could you please either specify in the comment that we are discarding the final (i.e. current) subinterval, or explicitly say `.pop(-1)`?

Reply:
see commit fb5552b. The word 'current' was added to the comment description and `.pop(-1)` was added for clarity.